src/data/analyse_en.json

```diff
@@ -4,10 +4,9 @@
 	"topic": "Analysis",
     "title": "Corona-Warn-App (CWA): Key performance indicators",
     "update": "Last update of the data:",
-    "updateNotice": "The CWA dashboard data is usually updated daily at 10:30 AM (CET) for the previous day.",
     "updateNoticeIcon": {
         "title": "Update of the data",
-        "content": "Downloads data is updated only on working days. On Mondays, the data for Friday, Saturday and Sunday are updated."
+        "content": "The CWA dashboard data is usually updated daily at 10:30 AM (CET) for the previous day. Downloads data is updated only on working days. On Mondays, the data for Friday, Saturday and Sunday are updated."
```

Comment:
I suggest instead of

"Downloads data ..."

change to

"Data regarding downloads ..."

This sounds better in English.

Reply:
Done, thank you!